freeipa.spec.in

```diff
@@ -35,6 +37,9 @@
 %if 0%{?fedora} >= 29
     # F29 only supports Python 3 as default Python
     %global with_default_python 3
+    %if 0%{?fedora} >= 30
+        %global with_python2 0
```

Comment:
This change hasn't been accepted yet. We need to provide Python 2 packages until Fedora has accepted removal of Python 2 packages.